### DISCUSSION TITLE
Balances when Pool not used

### DISCUSSION THREAD

**Discussion by saki2fifty** [RainbowMiner/RainbowMiner#1687] [Q&A] [answered]
Is there an option to see a pools balance when not selected?  Or to go through all pools for a balance?

I turn off Nicehash / others because I want to target just Eth from other pools, but would like to still see those disabled pools balances.

**Comment by RainbowMiner** [comment, marked as answer]
You could try to set `"ShowPoolBalancesExcludedPools": "1",` in config.txt. It instructs RainbowMiner to call all balance modules.

But be careful - if you make use of the "EnableAutoPool"/"EnableAutoCoin" feature, you will have a lot of failed API calls.

**Comment by saki2fifty** [comment]
Ok, I guess I missed this.  Just tested and I do see how balances now.  I will probably turn it on just to see my balances, then back off.

I think this works.  Thx!
